src/snowflake/cli/_plugins/nativeapp/entities/application.py

```diff
@@ -844,7 +844,7 @@ def get_events(
             raise ValueError("first and last cannot be used together")
 
         account_event_table = cls.get_account_event_table()
-        if not account_event_table:
+        if not account_event_table or account_event_table == "NONE":
```

Comment:
Discussed offline, this looks odd. Is the account param bad or is `NONE` truly a special value?

Reply:
Resolving, this is a supported value to mean event collection is disabled.

Reply:
Discussed offline #2: NONE is a valid value that we have to handle